training/02_Getting_Started/05_config_env_azure.adoc

```diff
@@ -0,0 +1,75 @@
+=== Workstation setup for Azure
+
+==== Prerequisites
+
+This is a configuration guide of the packages and repos you need to install the tools, but a previoulsy existing Azure account is needed.
+
+==== Steps
+
+If you want to deploy on azure you will need the Azure client.
+
+https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest[Source documentation]
+
+.in a nutshell (tested on fedora 28) - Azure cli (system-wide)
```

Comment:
Why are we publishing this tested on F28, which is 4 versions old, instead of RHEL 7 or RHEL 8? Is this our documentation or just pulled in from something else?

Reply:
cause I'm just writing the skel for the available versions we have today.

Reply:
In 2020 It needs to be RHEL8.